raiden/api/rest.py

```diff
@@ -289,11 +294,48 @@ def patch_channel(self, channel_address, balance=None, state=None):
                 )
                 return self.channel_schema.dumps(channel_to_api_dict(raiden_service_result))
             else:
+                # should never happen, channel_state is validated in the schema
                 return make_response(
                     'Provided invalid channel state {}'.format(state),
                     httplib.BAD_REQUEST,
                 )
 
+    def token_swap(
+            self,
+            identifier,
+            role,
+            sending_token,
+            sending_amount,
+            receiving_token,
+            receiving_amount):
```

Comment:
Added the newline, but guess since it comes right after line 310 it does not show up as addressed here.